addon/templates/components/frost-modal-dialog.hbs

```diff
@@ -71,6 +71,18 @@
     onClick=onCancel
   }}
 
+  {{#each params.buttons as |button index|}}
+    {{frost-button
+      hook=(concat hook '-button-' index)
```

Comment:
Why not make this part of the outer interface?  A specific name would be better than an index (which also could be brittle)